boxwise_flask/models.py

```diff
@@ -24,5 +25,54 @@ def __unicode__(self):
 
     @staticmethod
     def get_camps():
-        # SELECT distinct id, organisation_id FROM dropapp_dev.camps;
         return Camps.select().order_by(Camps.name)
+
+
+class Cms_Usergroups_Camps(db.Model):
+    camp_id = CharField()
+    cms_usergroups_id = CharField()
+
+    class Meta:
+        primary_key = CompositeKey('camp_id', 'cms_usergroups_id')
```

Comment:
Can you add a comment that we have to use a composite key due to the lack of a primary key in the original Mysql table scheme?

Reply:
@pylipp @mcgnly We do have a primary key for` cms_users`, `camps`, and `cms_usergroups`. The relationship is `cms_users `links to `cms_usergroups`, which links to `organisations` which links to `camps`.

Reply:
Possible that we were mistaken because the ERD shows that `Cms_Usergroups_Camps` has no primary key.

Reply:
@pylipp this is a good point which I also discussed with @mcgnly; not great if none of the cross-reference tables have no primary keys.  Have created a Trello card to track this issue. https://trello.com/c/B37cY4aN